Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -28,7 +27,7 @@ void Processor::initialize(u32 cpu)
     Prekernel::drop_to_exception_level_1();
 
     // Load EL1 vector table
-    el1_vector_table_install(&vector_table_el1);
+    Kernel::Aarch64::Asm::el1_vector_table_install(&vector_table_el1);
```

Comment:
Same here, we're already in the Kernel namespace

Reply:
Fixed!